### PR TITLE
D8CORE-562: Primary & Secondary Menus

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Changes Stanford Basic Branch
- Allows for all levels of the main navigation to be rendered out (trimming is handled for secondary differently.)

# Review By (Date)
- As soon as possible. 

# Urgency
- High, would like for testing next week.

# Steps to Test

1. Review the other PRs for tests
TBD

### CODE REVIEW

**Comment by pookmish** [comment]
The changes in stanford_basic break the tests due to the region selector: https://github.com/SU-SWS/stanford-caravan/blob/8.x-1.x/config/behat.yml#L84

**Comment by sherakama** [comment]
@pookmish 

This is ready for review again. 

https://navigation-test.sites.stanford.edu/

File: modules/stanford_profile_styles/css/stanford_page.css [hunk 1]
```diff
@@ -5,3 +5,14 @@
   max-width: 980px;
   margin: auto auto;
 }
+
+@media only screen and (max-width: 992px) {
+  .basic-page.node--layout-full .layout-pacman--left > div > aside {
```
Comment (pookmish) [inline modules/stanford_profile_styles/css/stanford_page.css]: we can remove these since we're removing pacman from the layouts

Comment (sherakama) [reply modules/stanford_profile_styles/css/stanford_page.css]: Yeah, I'll have to revise now.

**Pull request merged**